src/ripple/app/main/LoadManager.cpp

```diff
@@ -72,22 +85,24 @@ class LoadManagerImp
     //
     //--------------------------------------------------------------------------
 
-    void onPrepare ()
+    void onPrepare () override
     {
     }
 
-    void onStart ()
+    void onStart () override
     {
-        m_journal.debug << "Starting";
-        startThread ();
+        JLOG(mJournal.debug) << "Starting";
+        assert (! mThread.joinable());
+
+        mThread = std::thread {&LoadManagerImp::run, this};
```

Comment:
I was of two minds about this.  I vaguely remember Howard mentioning that with brace initialization in situations like this you can visually tell that it's not a function call.  That's why I chose braces in this circumstance.  But, again, I don't feel strongly about it so I'll change it.